fair/run.py

```diff
@@ -161,8 +163,10 @@ def run_command(
         _cfg['run_metadata']['script'] = bash_cmd
         yaml.dump(_cfg, open(config_yaml, 'w'))
 
-    _run_executable = "script" in _cfg["run_metadata"] or "script_path" in _cfg["run_metadata"]
-    _run_executable = _run_executable and mode == CMD_MODE.RUN
```

Comment:
As this checks if the `mode` is `CMD_MODE.RUN` all you need to do is simply set it to be another mode and this will work the same way, I would suggest just making a `CMD_MODE.PASS` or something. I can do that if you like.